cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case2b.prm

```diff
@@ -0,0 +1,119 @@
+# This cookbook models oceanic subduction in a 2d
+# box of 3000 x 670 km.
+# The subducting plate is driven by inflow boundary
+# conditions on the top part of the right boundary,
+# while a balancing outflow is prescribed underneath.
+# Only density and viscosity differ per composition.
+# Building on case 1, an initial temperature field
+# based on the plate cooling model and
+# a mantle adiabatic gradient of 0.25 degrees/km
+# is prescribed, but temperature still does not affect
+# the material parameters.
+# High resolution is required to a.o. resolve the weak zone
+# between the subducting and overriding plate, so about
+# 100 MPI processes are required.
+
+# We included a shared library for using the initial temperature plugin and the isotherm depth, composition RMS temperature, trench location postprocessors.
+set Additional shared libraries            = /home/bbkyingy/software/aspect_dealii_9.4.0/aspect/cookbooks/kinematically_driven_subduction_2d/libsubduction_plate_cooling.so
+
+# 2D 
+set Dimension                              = 2
+
+# Reuse parts of case 1
+include kinematically_driven_subduction_2d_case1.prm
+
+set Output directory                       = output-Case2b
+
+
+# We fix temperature on the top and bottom
+# as well as on the right boundary because 
+# we have inflow through the latter.
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top, right 
+  set List of model names = initial temperature 
+end
+
+
+subsection Material model
+  set Model name = multicomponent
+
+  subsection Multicomponent
+    # TODO the benchmark uses different reference temperatures for each composition
+    #set Reference temperature         = 1573.15,473.15,473.15,473.15,473.15,1573.15,1573.15,473.15 
+    set Reference temperature         = 473.15
+    set Viscosity averaging scheme    = maximum composition 
+    set Viscosities                   = 1.e20, 1.e23, 1.e20, 1.e23, 1.e23, 1.e23, 1.e23, 1.e20
+    set Densities                     = 3290.66,3000.0,3000.0,3250.0,3250.0,3290.66,3290.66,3200.0
+    # We change the reference densities for mantle and mantle lithosphere in order to achieve similar 
+    # densities when the reference temperature is changed from 1573.15K to 473.15K   
```

Comment:
```suggestion
    # densities when the reference temperature is set to 473.15K  instead of 1573.15K.
```
Can you move these two comment lines to above the `Densities` line?